packages/ckeditor5-dev-changelog/src/utils/normalizeentry.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import type { ParsedFile } from '../types.js';
+
+export function normalizeEntry( entry: ParsedFile, singlePackage: boolean ): ParsedFile {
+	// Normalize type.
+	const typeCapitalized = capitalize( entry.data.type );
+	const typeNormalized = typeCapitalized === 'Fixes' ? 'Fix' : typeCapitalized;
+
+	// Normalize breaking-change.
+	const breakingChange = entry.data[ 'breaking-change' ];
+	const breakingChangeLowerCase = String( breakingChange ).toLowerCase();
+	let breakingChangeNormalized: 'minor' | 'major' | boolean | undefined;
+
+	if ( singlePackage ) {
+		if ( breakingChangeLowerCase === 'true' ) {
+			breakingChangeNormalized = true;
+		} else if ( breakingChangeLowerCase === 'minor' ) {
+			breakingChangeNormalized = 'minor';
+		} else if ( breakingChangeLowerCase === 'major' ) {
+			breakingChangeNormalized = 'major';
+		}
+	} else {
+		if ( breakingChangeLowerCase === 'minor' ) {
+			breakingChangeNormalized = 'minor';
+		} else if ( breakingChangeLowerCase === 'major' ) {
+			breakingChangeNormalized = 'major';
+		}
+	}
+
+	// Normalize scope.
+	const scope = entry.data.scope;
+	const scopeNormalized = scope?.map( scopeEntry => String( scopeEntry ).toLowerCase() );
+
+	// Normalize closes.
+	const closesNormalized = entry.data.closes?.map( closes => String( closes ) );
+
+	// Normalize see.
+	const seeNormalized = entry.data.see?.map( see => String( see ) );
+
+	return {
+		...entry,
+		data: {
+			...entry.data,
+			typeNormalized,
+			breakingChangeNormalized,
+			scopeNormalized,
+			closesNormalized,
+			seeNormalized
```

Comment:
Couldn't we drop the _normalized_ suffix from properties' names? It looks weird.

Reply:
Well, we need them to include original, user-provided input in the validation messages. Let's see this example:
```js
{
  type: 'FIXES',
  typeNormalized: 'Fix'
}
```
The validation message will look like this: 
```
 - Type "FIXES" should be one of: "Feature", "Other" or "Fix" ("Fixes" is allowed) (case insensitive).
```

In order to get rid of `*Normalized` suffix, we would have to override the original data (e.g. override `type` field). This is not a bad idea, but it could be confusing, to understand in the future. And we would loose the ability to reference the unmodified data in the future.

Reply:
I am unsure if we need to display the invalid value. It's enough that we say "the provided one isn't supported". Update to xyz.

Reply:
Ok, I'm going to stop using `Normalized` and `Validated` suffixes, but I'm going to modify the original data.